### ISSUE TITLE
Blitzortung Custom Component Support

### ISSUE THREAD

**Issue by Norien** [ironsheep/lovelace-lightning-detector-card#3]
Is it possible to use this custom component with your card? - https://github.com/mrk-its/homeassistant-blitzortung

**Comment by ironsheep** [comment]
This is an interesting thought. But not really a good match. for at least two reasons

1. The ring data for the AS3935 sensor is accumulated outside of Home Assistant by the Daemon talking with the AS3935 sensor.  
1. This sensor does not provide any directional data, hence this card does not show any information about detection-direction from the center, only distance.

Given these two issues this doesn't seem to be a good fit.   Thoughts?

**Comment by casperleenheer** [comment]
I wanted the same. As I saw the answer I came up with this:

https://community.home-assistant.io/t/blitzortung-showing-thunder-on-map/214560

**Comment by ironsheep** [comment]
Thanks for posting this!  Additionally should one want to create a data source for this card it will be fairly easy with the guidance of a new document to be posted to the sensor daemon script repo. I'll post a final comment here when that doc. is committed to the repo.

**Comment by ironsheep** [comment]
Here's the latest document in the repo - a theory of operation which shows the payloads and accumulation logic/reasoning behind them.
See https://github.com/ironsheep/lightning-detector-MQTT2HA-Daemon/blob/master/THEOPS.md

It also describes how much of the work is already done for supporting another source of data.

**Comment by aaamoeder** [comment]
I came here to ask this too.. would be awesome !
directional data is nice to have but not essential imo and all the rest of the data is available so it should be possible ?

**Comment by ironsheep** [comment]
@aaamoeder yes is should be. See my https://github.com/ironsheep/lightning-detector-MQTT2HA-Daemon/blob/master/THEOPS.md for guidance.
